webknossos/webknossos/dataset/view.py

```diff
@@ -531,14 +641,39 @@ def read_xyz(
         relative_bounding_box: Optional[NDBoundingBox] = None,
         absolute_bounding_box: Optional[NDBoundingBox] = None,
     ) -> np.ndarray:
-        """
-        The user can specify the bounding box in the dataset's coordinate system.
-        The default is to read all data of the view's bounding box.
-        Alternatively, one can supply one of the following keyword arguments:
-        * `relative_bounding_box` in Mag(1)
-        * `absolute_bounding_box` in Mag(1)
-
-        Returns the specified data as a `np.array`.
+        """Read data from the view in XYZ coordinate order.
+
+        This method reads data from the view and returns it with dimensions ordered
+        as (channels, X, Y, Z) instead of the default (channels, Z, Y, X). This is
+        useful when working with tools or libraries that expect XYZ ordering.
+
+        Args:
+            relative_bounding_box (Optional[NDBoundingBox], optional): Bounding box relative
+                to view's position in Mag(1) coordinates. Defaults to None.
+            absolute_bounding_box (Optional[NDBoundingBox], optional): Absolute bounding box
+                in Mag(1) coordinates. Defaults to None.
+
+        Returns:
+            np.ndarray: The requested data as a numpy array with dimensions ordered as
+                (channels, X, Y, Z) for multi-channel data or (X, Y, Z) for single-channel
+                data. Areas outside the dataset are zero-padded.
+
+        Examples:
+            ```python
+            # Read entire view's data in XYZ order
+            view = layer.get_mag("1").get_view(size=(100, 100, 10))
+            xyz_data = view.read_xyz()  # Returns (X, Y, Z) array
+
+            # Read with relative bounding box
+            bbox = NDBoundingBox((10, 10, 0), (50, 50, 10))
```

Comment:
NDBoundingBox constructor needs additional arguments e.g.:`axis=("x", "y", "z")` and `index=(1, 2, 3)`.